lib/utils/eol-conversion.ts

```diff
@@ -111,7 +92,7 @@ export async function readFileWithEolConversion(
 	}
 
 	// Skip further processing of files that don't contain CRLF
-	if (!fileBuffer.includes('\r\n', 0, encoding)) {
+	if (!fileBuffer.includes('\r\n')) {
```

Comment:
The `encoding` argument to `includes()` refers to the encoding of the `'\r\n'` string, not the encoding of the string in the `fileBuffer`.

Reply:
Ah, nice catch.  (Though I think a side-effect-less mistake)